documentation/modules/con-kafka-connect-s2i-logging.adoc

```diff
@@ -0,0 +1,54 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connectS2I.adoc
+
+[id='con-kafka-connect-s2i-logging-{context}']
+= Kafka Connect with S2I loggers
+
+Kafka Connect with Source2Image support has its own configurable loggers:
+
+* `connect.root.logger.level`
+* `log4j.logger.org.apache.zookeeper`
```

Comment:
Same as above for the non-S2I implementation.

Reply:
Removed